client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/styles.scss

```diff
@@ -254,6 +254,7 @@
 				display: inline-block;
 				margin-bottom: 16px;
 				white-space: nowrap;
+				line-height: 24px;
```

Comment:
there's a lot of changes here that would be good to componetize so we don't run into them over and over

Reply:
Logged this as a card in our board so we can break it out separate.